Content.Shared/Armor/SharedArmorSystem.cs

```diff
@@ -29,6 +31,15 @@ private void OnDamageModify(EntityUid uid, ArmorComponent component, InventoryRe
         args.Args.Damage = DamageSpecifier.ApplyModifierSet(args.Args.Damage, component.Modifiers);
     }
 
+    private void OnChangelingDevourAttempt(EntityUid uid, ArmorComponent component, InventoryRelayedEvent<ChangelingDevourAttemptEvent> args)
+    {
+        var slash = component.Modifiers.Coefficients.ContainsKey("Slash") && component.Modifiers.Coefficients["Slash"] < 1f - args.Args.ProtectionThreshold;
+        var blunt = component.Modifiers.Coefficients.ContainsKey("Blunt") && component.Modifiers.Coefficients["Blunt"] < 1f - args.Args.ProtectionThreshold;
+        var pierce = component.Modifiers.Coefficients.ContainsKey("Piercing") && component.Modifiers.Coefficients["Piercing"] < 1f - args.Args.ProtectionThreshold;
+
+        args.Args.Protection = slash || blunt ||  pierce;
+    }
+
```

Comment:
I know this is still in draft but anything ling related shouldn't be in the armor system. What are you trying to do?

Reply:
This was brought up in Developer and is being changed to be a query of some form for the coefficents. 

Was caused as a failure to understand what InventoryRelays do and how to make one, and has been clarified (will be the next commit i do)